### ISSUE TITLE
Add DOI and ArXiv articles to results summary.

### ISSUE THREAD

**Issue by marshalmiller** [marshalmiller/rottingresearch#43]
It would be amazing to add number of DOI links and ArXiv references to the link summary page.

**Comment by marshalmiller** [comment]
Here are the arguments from the Python script.

        arxiv = extractor.extract_arxiv(uri)
        if arxiv:
            self.ref = arxiv.pop()
            self.reftype = "arxiv"
            return

        doi = extractor.extract_doi(uri)
        if doi:
            self.ref = doi.pop()
            self.reftype = "doi"
            return

**Comment by marshalmiller** [comment]
@aditirao7 Right now it looks like the analysis page is generated solely by code returned. Does that mean that this change would require a whole new process to pull URL type?

**Comment by aditirao7** [comment]
I think I can take this up, could you share the pdf you used to test this @marshalmiller?

**Comment by marshalmiller** [comment]
@aditirao7 This should work. Has DOI and ArXiv references. If you notice, ArXiv results are omitted from the URL results. That is one of the reasons I wanted this issue handled. The program returns the ArXiv and DOIs differently.

https://arxiv.org/pdf/2206.00785.pdf

**Comment by aditirao7** [comment]
> @aditirao7 Right now it looks like the analysis page is generated solely by code returned. Does that mean that this change would require a whole new process to pull URL type?

Yes, the analysis page is being generated based on whatever is returned from the linkrot library. I managed to add the number of Arxiv and DOI refs in the summary part along with the extra sections for both references. Since there are DOI refs in this pdf that aren't showing up I think another issue should be opened in the linkrot repo? Let me know if I should push this code though. Here's how it looks right now:
 
![Screenshot from 2022-06-27 17-00-28](https://user-images.githubusercontent.com/57457307/175931709-80f1bd9b-41a2-45f5-8f74-513186254f53.png)
![Screenshot from 2022-06-27 17-00-42](https://user-images.githubusercontent.com/57457307/175931726-302e0331-49e8-4042-b50b-f68ee7aa9483.png)

Random sidenote: I've gotta say the website looks great 🤩, it's been a while since I had a chance to look at the repo because I've been swamped with coursework but it's nice to see how far the website has come. I learned a bunch of new things while going over the new codebase too. I'll make sure to contribute whenever I get the time :)

**Comment by marshalmiller** [comment]
@aditirao7 Yes. If you could push it, that would be great. Yeah. I'm very happy with how it's coming a long. We've had a lot of great contributions. Thank you for your help.

**Comment by aditirao7** [comment]
Sure, I'll send in the PR soon. There's also a few bugs I noticed, can I open issues for them?

**Comment by marshalmiller** [comment]
@aditirao7 Yes. Please open as many issues as you'd like. I'd love to get all the bugs worked out. Thanks, again.
